pkgs/applications/networking/cluster/osm/cli/default.nix

```
@@ -0,0 +1,33 @@
+{ buildGoModule, fetchFromGitHub, stdenv }:
+
+let
+  version = "0.1.0";
+  sha256 = "0hc63xvww4p4bqsq2ihz5in5ia9x7ibfa7ixd6fkxvdb1rbwxrx6";
+  vendorSha256 = "03qy4gl9lii4amsfg2y4ib5xwg74gri601pkfa3c7jwc034xf5kf";
+in buildGoModule {
+  name = "osm-cli-${version}";
+  pname = "osm";
+  version = version;
+
+  vendorSha256 = vendorSha256;
```

```suggestion
  inherit version vendorSha256;

  pname = "osm";
```

`name` is defined in terms of `pname` and `version` in `stdenv`, so you do not need to define it explicitly.